hooks/useNotificationManager.ts

```diff
@@ -0,0 +1,72 @@
+import { useAccount, useProvider } from "@starknet-react/core";
+import { useAtom } from "jotai";
+import { atomWithStorage } from "jotai/utils";
+import { useEffect } from "react";
+import { hexToDecimal } from "../utils/feltService";
+import { NotificationType } from "../utils/constants";
+
+const notificationsAtom = atomWithStorage<SIDNotification<NotificationData>[]>(
+  "userNotifications_SID",
+  []
+);
```

Comment:
Can you explain briefly to be sure I understand what does jotai package does

Reply:
So `jotai` manages for us the storage, update and reading of the local storage. It ensures that the data we're passing is of type  `<SIDNotification<NotificationData>[]>`. In local storage it stores the data in `userNotifications_SID`. The hook `useAtom` acts similarly to a `useState` (line 16) so we can update the data we store in local storage and access it in the `useNotificationManager` hook.

Reply:
super thx !